arangod/Aql/ExecutionBlockImpl.h

```diff
@@ -135,6 +139,9 @@ class ExecutionBlockImpl final : public ExecutionBlock {
     DONE
   };
 
+  static constexpr bool isParallelExecutor =
```

Comment:
Why do you need this static inside the header?

Reply:
I don't (not anymore, anyhow).

Reply:
I've already moved it in the `.cpp` file in the follow-up branch; but can apply that here to avoid the diff here and later.